src/components/announcements/IntroProjectTags.tsx

```diff
@@ -1,10 +1,27 @@
+import { Trans } from '@lingui/macro'
+import { ProjectTagsRow } from 'components/ProjectTagsRow'
+import { projectTagOptions } from 'models/project-tags'
+import Link from 'next/link'
+
 export default function IntroProjectTags() {
```

Comment:
new in-app announcement for project tags, replacing previous placeholder announcement. this will be shown on a project page when the owner wallet is connected